### PR TITLE
Release Cohort Views

### PR DESCRIPTION
**Pull request by Br4nd0R**
## Added
* New cohort list and detail views at `/cohorts` and `/cohorts/{cohort.id}` respectively. These views are only accessible when users are logged in _and_ they have the 'view release cohorts' permission.

## For QA:
1. Create a release cohort record
2. Navigate to `/cohorts` to see list view
3. Click on the cohort on the list view to navigate to the cohort detail page.
4. Observe message present when no products are in a release cohort
5. Create a release object and associate it with the new release cohort.
6. Return to the detail view, to see the product is now included in the view.
7. Log out and try to access either view, you should be redirected to login.

## Misc.
* I created a story to enable these views to be accessed somewhere in the UI (instead of direct via URL only).
* We may want a story to optimize the list view. As the number of cohort records grows, we will likely want a way to show the "upcoming" or "active" release.
* Thoughts on if people like the "merge freeze active" to be green when _not active_?
* Do we want additional/different columns for products releasing on the detail view? 
* Do we want additional/different columns for the list view?

## Screenshots

`cohort_list`
cohort_list

`cohort_detail`
cohort_detail

### CODE REVIEW

File: metaci/release/templates/cohort/detail.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
+{% extends 'layout_full.html' %}
+
+<!-- TODO: link back to /cohorts -->
```
Comment (Br4nd0R) [inline metaci/release/templates/cohort/detail.html:3]: I haven't forgot about this.

Comment (Br4nd0R) [reply metaci/release/templates/cohort/detail.html:3]: Or 💭 ... maybe the story for adding navigation to the list view could include making a "Releases" tab on MetaCI's header that only people with the view cohorts perm could see.

File: metaci/release/templates/cohort/detail.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+<div class="slds-text-heading_small">What's a Merge Freeze?</div>
+<div class="slds-box">
+  <p>
+    A merge freeze is a period of time where repositories are restricted to not
+    merging any new code into their main branches.
+    This allows release engineering and the product teams to ensure that no untested changes are inadvertently included in a product version that is released to customers.
+    When a merge freeze is active, all repositories listed below will 
+    have pull requests blocked by a failing check in GitHub titled "Merge Freeze".
+    When the merge freeze is lifted, these checks will be reran and will begin passing.
```
Comment (jstvz) [inline metaci/release/templates/cohort/detail.html:60]:
```suggestion
    When the merge freeze is lifted, these checks will rerun and pass.
```

File: metaci/release/templates/cohort/detail.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+{% block layout_body %}
+
+{% if releases %}
+
+<div class="slds-text-heading_small">What's a Merge Freeze?</div>
+<div class="slds-box">
+  <p>
+    A merge freeze is a period of time where repositories are restricted to not
+    merging any new code into their main branches.
+    This allows release engineering and the product teams to ensure that no untested changes are inadvertently included in a product version that is released to customers.
```
Comment (jstvz) [inline metaci/release/templates/cohort/detail.html:57]:
```suggestion
    This allows release engineering and product teams to ensure no untested changes are accidentally released to customers.
```

File: metaci/release/templates/cohort/detail.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+
+{% block layout_body %}
+
+{% if releases %}
+
+<div class="slds-text-heading_small">What's a Merge Freeze?</div>
+<div class="slds-box">
+  <p>
+    A merge freeze is a period of time where repositories are restricted to not
```
Comment (jstvz) [inline metaci/release/templates/cohort/detail.html:55]:
```suggestion
    A merge freeze is a period of time where repositories are blocked from
```

**Review by jstvz — APPROVED** [review]
Minor changes to the MF definition.

**Comment by Br4nd0R** [comment]
@jstvz - Feel free to merge this once you approve it.

**Review by jstvz — APPROVED** [review]

**Pull request merged**